composer.json

```diff
@@ -5,7 +5,7 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
-    "codeception/codeception": "^2.0.0 || ^3.0.0"
+    "codeception/codeception": "^2.0.0 || ^3.0.0 || ^4.0.0"
```

Comment:
I am wondering if 3 + 4 are sufficient, do we need to keep also v2? Or is it an easy win to keep it?

Reply:
IMO most of the projects are still on v2.